template/query.rs.template

```diff
@@ -22,4 +22,26 @@ impl<'a> VersionedRustdocAdapter<'a> {
             {{/each}}
         }
     }
+
+    pub fn run_query_with_indexed_query<'slf: 'a, K: Into<Arc<str>>, V: Into<FieldValue>>(
+        &'slf self,
+        query: Arc<trustfall_core::ir::IndexedQuery>,
+        vars: BTreeMap<K, V>,
+    ) -> anyhow::Result<Box<dyn Iterator<Item = QueryResult> + 'a>> {
+        let vars = Arc::new(vars.into_iter().map(|(k, v)| (k.into(), v.into())).collect());
```

Comment:
nit: try to match how the template gets formatted in the final code, to make the code easier to debug
```suggestion
        let vars = Arc::new(
            vars.into_iter()
                .map(|(k, v)| (k.into(), v.into()))
                .collect(),
        );
```